postgres/toggle-backup-activation.sh

```diff
@@ -94,4 +94,32 @@ else
     echo "" >> /etc/cron.d/backup_postgres_crontab
     service cron restart
     echo "PostgreSQL automatic backup schedule: ${POSTGRES_BACKUP_SCHEDULE}"
-fi
+
+    if [[ ${USE_WAL_E} -eq "$TRUE") ]]; then
```

Comment:
Good idea but kobo-install is only supporting `WAL-E` or `pg_dump`, not both at the same time
Thus, this block should be moved inside the `USE_S3` block.